plugins.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
The cadence to update the repo would be per release, how about grouping per release and then it includes the plugins under it ? This way we'll have to add only one block (of release) per release than updating multiple.
```
---
v0.22.0:
   source-kafka:
     [...]
   admin:
     [...]
```

Reply:
OK, but in this case we end up repeating the `homepage` in  each version...  I suppose a small price to pay. I could make it the download so it's pegged to the version but the homepage is also quite useful. I'll figure something